jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/textfield/JmixBigDecimalField.java

```diff
@@ -51,6 +55,11 @@ public void setApplicationContext(ApplicationContext applicationContext) {
     @Override
     public void afterPropertiesSet() {
         initComponent();
+
+        messages = applicationContext.getBean(Messages.class);
+
+        fieldDelegate.addValidator(new NumberParseValidation<>(
```

Comment:
Even though, the issue refers to BigDecimal field, `JmixIntegerField` have the same issue. Probably, other type specific fields too. 

The more universal solution is to handle client-side value presence (without specific `NumberParseValidation`) in the field delegate. We have `AbstractNumberFieldDelegate` already taht is used by IntegerField and NumberField.

Reply:
<img width="678" alt="Screenshot 2024-06-10 at 16 26 55" src="https://github.com/jmix-framework/jmix/assets/6077149/76dd461b-c839-472d-a36f-9d253663cdb9">